file.go

```diff
@@ -1084,6 +1084,20 @@ func (f *File) FlattenBatches() (*File, error) {
 			}
 		}
 	}
+	for i := range out.Batches {
+		bh := out.Batches[i].GetHeader()
+		if bh.StandardEntryClassCode != "ADV" {
+			batch, _ := NewBatch(bh)
+			entries := sortEntries(out.Batches[i].GetEntries())
```

Comment:
We should update the GoDoc to reflect this too

Reply:
Fixed, thanks!